websites/V/Vodafone FM/presence.ts

```diff
@@ -0,0 +1,33 @@
+const presence = new Presence({
+		clientId: "1061398473368412210",
+	}),
+	strings = presence.getStrings({
+		play: "general.playing",
+		pause: "general.paused",
+		listeningMusic: "general.listeningMusic",
+	});
```

Comment:
If you're using multiLang, you must have a setting for this as shown: 
https://github.com/PreMiD/Presences/blob/e9f2b3e706d981782d7421317a42d9dd8cadfbc3/websites/N/Netflix/metadata.json#L68-L72

Strings should be initialised like
https://github.com/PreMiD/Presences/blob/e9f2b3e706d981782d7421317a42d9dd8cadfbc3/websites/N/Netflix/presence.ts#L30-L55

And the respective type so you do not have to do `(await strings)` over and over
https://github.com/PreMiD/Presences/blob/e9f2b3e706d981782d7421317a42d9dd8cadfbc3/websites/N/Netflix/presence.ts#L65

Reply:
Thanks for the tip! Addressed in https://github.com/PreMiD/Presences/pull/7113/commits/70716226d55f55bae1bc5553b4c2f92d065cd27d